src/manual/tasks/workertypes.md

```diff
@@ -0,0 +1,17 @@
+---
+filename: tasks/workertypes.md
+title: Worker Types
+order: 20
+---
+
+Workers execute tasks, but there are many types of workers. A task specifies a
+single "worker type" by which it should be executed, using the two-part
+namespace `<provisionerId>/<workerType>`.  Workers with the same worker type
+all consume tasks from a single queue, as [described
```

Comment:
I added this:
```
Workers execute tasks, but there are many types of workers. A task specifies a
single "worker type" by which it should be executed, using the two-part
namespace `<provisionerId>/<workerType>`.  The naming of the identifiers can be
a bit confusing: when we refer to a worker type, it is to the combination of
both identifiers. Thus `gcp-provisioner/persona-build` and
`rackspace-provisioner/persona-build` are completely different worker types,
despite sharing the same `workerType` identifie
```
hopefully that clarifies it? I agree it is an unfortunate choice of identifier names..